### PR TITLE
arch sucks

### PR DESCRIPTION
**Pull request by Nik-Sch**

### CODE REVIEW

File: shared_dotfiles/change_theme.py [hunk 1]
```diff
@@ -54,17 +54,17 @@ def set_vscode(name: str):
     settings_path.write_text(json.dumps(settings, indent=4))
 
 
-def go_dark():
+def go_dark(background_path: str):
```
Comment (g0t00) [inline shared_dotfiles/change_theme.py:57]: why not path thingy?

File: shared_dotfiles/detect_display.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    parser = argparse.ArgumentParser(
+        prog="detect_display",
+        description="detects displays and enables them accordingly",
+    )
+    parser.add_argument("-b", "--background-path", default=Path.home())
+    parser.add_argument("-d", "--desktops", nargs="+")
+    args = parser.parse_args()
+
+
+    # be careful: has side effects
```
Comment (g0t00) [inline shared_dotfiles/detect_display.py:22]: just not inplace sort eh?

File: shared_dotfiles/lock.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        "--modif-color": "d23c3dff",
+        "--noinput-text": "",
+        "--keylayout": "0",
+        "--screen": "$display_on",
+    }
+
+    subprocess.call(
+        [
+            "i3lock",
+            *itertools.chain(*[[key, value] for key, value in params.items()]),
```
Comment (g0t00) [inline shared_dotfiles/lock.py:97]: whut? what is this? flat?

File: shared_dotfiles/change_theme.py (hunk 1 above)
Comment (Nik-Sch) [reply shared_dotfiles/change_theme.py:57]: i didn't change the logic

File: shared_dotfiles/detect_display.py (hunk 2 above)
Comment (Nik-Sch) [reply shared_dotfiles/detect_display.py:22]: didn't change the logic

File: shared_dotfiles/lock.py (hunk 3 above)
Comment (Nik-Sch) [reply shared_dotfiles/lock.py:97]: don't know, python sucks

**Review by g0t00 — APPROVED** [review]

**Pull request merged**